LandSandBoat/server
# [Item Mods] Update all Double/Triple Attack damage gear

**_I affirm:_**

## What does this pull request do?

Updates all equipment that uses the DOUBLE_ATTACK_DMG and TRIPLE_ATTACK_DMG mods created in #2080

Affects the following gear:
**Double Attack damage** - Widowmaker, Warrior’s Cuisses +2, Devivifier, Agoge Cuisses, Agoge Cuisses +1, Agoge Cuisses +2, Agoge Cuisses +3, Cichol’s Mantle, Quiahuiz Helm, Quiahuiz Trousers
**Triple Attack damage** - Demonry Ring, Triplus Dagger, Assassin's Poulaines +2, Chiner's Belt, Chiner's Belt +1, Plunderer's Poulaines, Plunderer's Poulaines +1, Plunderer's Poulaines +2, Plunderer's Poulaines +3, Assassin's Gorget, Assassin's Gorget +1,  Assassin's Gorget +2, Hetairoi Ring, Toutatis's Cape



## Steps to test these changes

Update the local database with the item_mod.sql
Add the equipment to your character
Whack some mobs and see if your double/triple attack white damage increases from your normal white damage

Useful commands:
!setmod 1038 1-100 charName (Sets DOUBLE_ATTACK_DMG to specified value)
!setmod 1039 1-100 charName (Sets TRIPLE_ATTACK_DMG to specified value)
!setmod 288 1-100 charName (sets DOUBLE_ATTACK proc rate to specified value)
!setmod 302 1-100 charName (sets TRIPLE_ATTACK proc rate to specified value)
